app/src/main/java/ch/hikemate/app/ui/auth/SignInWithEmailScreen.kt

```diff
@@ -0,0 +1,148 @@
+
+  Column(
+      modifier =
+          Modifier.testTag(Screen.SIGN_IN_WITH_EMAIL)
+              .padding(
+                  // Add padding to the sidebar padding
+                  start = 16.dp,
+                  end = 16.dp,
+                  top = 16.dp,
+              ),
+      verticalArrangement = Arrangement.spacedBy(16.dp)) {
+        BackButton(navigationActions)
+        Text(
+            context.getString(R.string.SIGN_IN_WITH_EMAIL_TITLE),
+            style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 32.sp),
+            modifier = Modifier.testTag(SignInWithEmailScreen.TEST_TAG_TITLE))
+
+        OutlinedTextField(
+            modifier = Modifier.fillMaxWidth().testTag(SignInWithEmailScreen.TEST_TAG_EMAIL_INPUT),
+            keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Email),
+            colors = inputColors,
+            value = email,
+            onValueChange = { email = it },
+            label = { Text(context.getString(R.string.SIGN_IN_WITH_EMAIL_EMAIL_LABEL)) })
+
+        OutlinedTextField(
+            modifier =
+                Modifier.fillMaxWidth().testTag(SignInWithEmailScreen.TEST_TAG_PASSWORD_INPUT),
+            visualTransformation = PasswordVisualTransformation(),
+            colors = inputColors,
+            value = password,
+            onValueChange = { password = it },
+            label = { Text(context.getString(R.string.SIGN_IN_WITH_EMAIL_PASSWORD_LABEL)) })
+
+        BigButton(
+            modifier =
+                Modifier.fillMaxWidth().testTag(SignInWithEmailScreen.TEST_TAG_SIGN_IN_BUTTON),
+            buttonType = ButtonType.PRIMARY,
+            label = context.getString(R.string.SIGN_IN_WITH_EMAIL_SIGN_IN_BUTTON),
+            onClick = {
+              authViewModel.signInWithEmailAndPassword(
+                  email,
+                  password,
+                  onSuccess = {
+                    // Navigate to the map screen
+                    navigationActions.navigateTo(Route.MAP)
+                  },
+                  onErrorAction = {
+                    // Show an error message in a toast
+                    Toast.makeText(context, it.message, Toast.LENGTH_SHORT).show()
+                  })
+            })
+
+        // Push the sign up button to the bottom of the screen by adding a spacer
+        Spacer(modifier = Modifier.weight(1f))
+
+        Box(
+            modifier = Modifier.fillMaxWidth().padding(bottom = 16.dp),
+            contentAlignment = Alignment.Center) {
+              TextButton(
+                  onClick = {
+                    // Navigate to the sign up screen
+                    // navigationActions.navigateTo(Screen.SIGN_UP_WITH_EMAIl)
+                  },
+                  modifier = Modifier.testTag(SignInWithEmailScreen.TEST_TAG_GO_TO_SIGN_UP_BUTTON),
+              ) {
+                Text(
+                    context.getString(R.string.SIGN_IN_WITH_EMAIL_GO_TO_SIGN_UP),
```

Comment:
Nitpick: use the composable function `stringResource` instead.

```suggestion
                    stringResource(R.string.SIGN_IN_WITH_EMAIL_GO_TO_SIGN_UP),
```